iOS/MMIWG2S/Views/ARFaceViewerUI.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  ARFaceViewerUI.swift
+//  MMIWG2S
+//
+//  Created by Callum Johnston on 10/25/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol ARFaceViewerUIDelegate {
+    func textureChanged(toRed: Bool)
+    func generateShareImage() -> UIImage?
+    func resumeSessionOnceCaptured()
+}
+
+class ARFaceViewerUI: UIView {
+    private let shareButtonBottomPadding: CGFloat = 50
+    private let shareButtonDiameter: CGFloat = 105
+    private let toggleViewWidth: CGFloat = 97
+    private let toggleViewBottomPadding: CGFloat = 75
+    private let largerColorToggleWidth: CGFloat = 54
+    private let smallerColorToggleWidth: CGFloat = 36
```

Comment:
❓ Would these be ok to move into the SceneButton class as static fields that can be called?

Reply:
Sure thing